helm/ovn-kubernetes/crds/policy.networking.k8s.io_adminnetworkpolicies.yaml

```diff
@@ -0,0 +1,1261 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
so if CRDs are stored here now as well...how will they be updated? Is it manual, or has it been added to our codegen scripts?

Reply:
most of the CRDs are sym linked to the original locations, except servicemonitor and promrule.

Reply:
note that ANP CRDs are becoming an online install: https://github.com/ovn-org/ovn-kubernetes/pull/4245/commits/b06f4a1ba3f71450b5242632af24cb0993a05198#diff-74a1a8552feeb3203caa0b2f9cc5c9c0eeca76451e37f1a61865c598b3247444R991 and not maintaining local copies, so this symlink will break if not kept up to date.

Reply:
Good to know! One idea @jxiaobin had on this is that we can enhance `kind-helm.sh` to do the curl, as we are not sure how to make helm use an external crd. If this PR merges before https://github.com/ovn-org/ovn-kubernetes/commit/b06f4a1ba3f71450b5242632af24cb0993a05198 , I can definitely help on making that change.